### DISCUSSION TITLE
The OpenTTD 1.11 Title Screen Competition

### DISCUSSION THREAD

**Discussion by 2TallTyler** [OpenTTD/OpenTTD#9606] [Title Screen Competition] [answered]
# The OpenTTD 1.11 Title Screen Competition
![image](https://openttd.github.io/title-screen-competition-1.11/layout/winner-1.10.png)

Hello everyone, and welcome to the 1.11 title screen competition!

This is your opportunity to design and help choose the savegame which plays behind the main menu of OpenTTD; pictured above is **ALEXbr**'s winning entry for last year's 1.10 release.

Voting has ended, and the final results are:
* Erato: 14
* Timberwolf: 25
* didac: 20
* Emperor Jake: 45
* Chrnan6710: 75

**Congratulations to Chrnan6710!**

The rules for the competition are as follows:

## Schedule

* Submission deadline: Sunday, March 14, 2021, at 23:59:00 UTC.
* Voting opens Monday, March 15, 2021.
* Voting closes: Sunday, March 28, 2021, at 23:59:00 UTC.

## Entry

All entries will be posted in [this topic](https://www.tt-forums.net/viewtopic.php?f=29&t=88509) as a savegame.
You may change your entry at any time, but only one entry will be accepted by the close of entries.
If updating, please post a new entry and either delete the old entry or PM a moderator to have it removed.

Entries are now closed.

## Voting

Voting will take place here on GitHub. You will need a GitHub account (free) to vote. Please see the entries below this post and **click the up arrow to the left of an entry to vote for it**. The number below the arrow shows the current vote total for that entry.

You may vote for multiple entries (although only once for each entry).

2TallTyler will refrain from voting except to cast a tiebreaker vote.

## Title screen guidelines

* This will be a first impression of the game for many new players, so create something which you feel summarizes the game. This is usually a busy, exciting scene showing off many modes of transport, but there are no rules about what must be shown.
* A traditional part of these title games is featuring [new features](https://cdn.openttd.org/openttd-releases/1.11.0-RC1/changelog.txt) of the release. Last year that was multiple docks per station. This year in 1.11, the most visible new feature is that towns can now build road tunnels both under obstructions and through mountains.
* A map size of 256 x 256 is ideal.
* The main menu viewport is set when you save the game, anchored at the top left corner and expanding down and to the right as screen sizes increase. Check out [last year's entries](https://www.tt-forums.net/viewtopic.php?f=29&t=86795) to see how a player's screen size affects their view of your entry. Please design both for small and large screens to give everybody an interesting view.
* Note that any signs you place will be visible.
* Breakdowns will automatically be disabled. Cheating for money is OK. Unprofitable companies are OK. Building with multiple companies is OK. Building the game in the Scenario Editor is OK.
* Do not use AIs, GameScripts, or NewGRFs. These are automatically disabled.
* The game must be created in a clean, master branch version of OpenTTD or its official binaries. The game cannot be created in custom modified versions of OpenTTD or in popular patch packs such as JGR Patch Pack, LtJoker's Patch Pack, Reddit client, CityMania client, etc.

Feel free to check out [last year's competition](https://www.tt-forums.net/viewtopic.php?f=29&t=86795) for inspiration.

Good luck!

**Comment by 2TallTyler** [comment]
# Erato

Toyland needs some more love. I did my best to try and make something that looked decent.
#### 1024x768
![poster](https://openttd.github.io/title-screen-competition-1.11/screens/Erato-1024_768.png)

[Image](https://openttd.github.io/title-screen-competition-1.11/screens/Erato-1024_768.png)
[Animated GIF](https://openttd.github.io/title-screen-competition-1.11/screens/Erato-1024_768.gif)
[Movie (low quality)](https://openttd.github.io/title-screen-competition-1.11/screens/Erato-1024_768.mp4)

#### 1280x1024
[Image](https://openttd.github.io/title-screen-competition-1.11/screens/Erato-1280_1024.png)
[Animated GIF](https://openttd.github.io/title-screen-competition-1.11/screens/Erato-1280_1024.gif)
[Movie (low quality)](https://openttd.github.io/title-screen-competition-1.11/screens/Erato-1280_1024.mp4)

#### 1440x900
[Image](https://openttd.github.io/title-screen-competition-1.11/screens/Erato-1440_900.png)
[Animated GIF](https://openttd.github.io/title-screen-competition-1.11/screens/Erato-1440_900.gif)
[Movie (low quality)](https://openttd.github.io/title-screen-competition-1.11/screens/Erato-1440_900.mp4)

#### 1920x1080
[Image](https://openttd.github.io/title-screen-competition-1.11/screens/Erato-1920_1080.png)
[Movie (low quality)](https://openttd.github.io/title-screen-competition-1.11/screens/Erato-1920_1080.mp4)

#### 2560x1440
![poster](https://openttd.github.io/title-screen-competition-1.11/screens/Erato-2560_1440.png)

[Image](https://openttd.github.io/title-screen-competition-1.11/screens/Erato-2560_1440.png)
[Movie (low quality)](https://openttd.github.io/title-screen-competition-1.11/screens/Erato-2560_1440.mp4)

**Reply by Drury27** [reply]
don't worry toylandchads, one day... one day we'll win

**Reply by ske2004** [reply]
my favorite unironically

**Comment by 2TallTyler** [comment]
# Timberwolf

Some hilly subtropical terrain, harking back to the original Transport Tycoon Deluxe title game, although desert rather than rainforest this time. I've aimed for some of the same feel with not making things too "busy" or trying to cram something into every tile, but working with the landscape and having a nice laid-back network while still showing some possibilities like RoRo stations for horizontal resolutions >1600px. And of course lots of town tunnels!
#### 1024x768
![poster](https://openttd.github.io/title-screen-competition-1.11/screens/Timberwolf-1024_768.png)

[Image](https://openttd.github.io/title-screen-competition-1.11/screens/Timberwolf-1024_768.png)
[Animated GIF](https://openttd.github.io/title-screen-competition-1.11/screens/Timberwolf-1024_768.gif)
[Movie (low quality)](https://openttd.github.io/title-screen-competition-1.11/screens/Timberwolf-1024_768.mp4)

#### 1280x1024
[Image](https://openttd.github.io/title-screen-competition-1.11/screens/Timberwolf-1280_1024.png)
[Animated GIF](https://openttd.github.io/title-screen-competition-1.11/screens/Timberwolf-1280_1024.gif)
[Movie (low quality)](https://openttd.github.io/title-screen-competition-1.11/screens/Timberwolf-1280_1024.mp4)

#### 1440x900
[Image](https://openttd.github.io/title-screen-competition-1.11/screens/Timberwolf-1440_900.png)
[Animated GIF](https://openttd.github.io/title-screen-competition-1.11/screens/Timberwolf-1440_900.gif)
[Movie (low quality)](https://openttd.github.io/title-screen-competition-1.11/screens/Timberwolf-1440_900.mp4)

#### 1920x1080
[Image](https://openttd.github.io/title-screen-competition-1.11/screens/Timberwolf-1920_1080.png)
[Movie (low quality)](https://openttd.github.io/title-screen-competition-1.11/screens/Timberwolf-1920_1080.mp4)

#### 2560x1440
![poster](https://openttd.github.io/title-screen-competition-1.11/screens/Timberwolf-2560_1440.png)

[Image](https://openttd.github.io/title-screen-competition-1.11/screens/Timberwolf-2560_1440.png)
[Movie (low quality)](https://openttd.github.io/title-screen-competition-1.11/screens/Timberwolf-2560_1440.mp4)

**Comment by 2TallTyler** [comment]
# didac

Here is mine!
#### 1024x768
![poster](https://openttd.github.io/title-screen-competition-1.11/screens/didac-1024_768.png)

[Image](https://openttd.github.io/title-screen-competition-1.11/screens/didac-1024_768.png)
[Animated GIF](https://openttd.github.io/title-screen-competition-1.11/screens/didac-1024_768.gif)
[Movie (low quality)](https://openttd.github.io/title-screen-competition-1.11/screens/didac-1024_768.mp4)

#### 1280x1024
[Image](https://openttd.github.io/title-screen-competition-1.11/screens/didac-1280_1024.png)
[Animated GIF](https://openttd.github.io/title-screen-competition-1.11/screens/didac-1280_1024.gif)
[Movie (low quality)](https://openttd.github.io/title-screen-competition-1.11/screens/didac-1280_1024.mp4)

#### 1440x900
[Image](https://openttd.github.io/title-screen-competition-1.11/screens/didac-1440_900.png)
[Animated GIF](https://openttd.github.io/title-screen-competition-1.11/screens/didac-1440_900.gif)
[Movie (low quality)](https://openttd.github.io/title-screen-competition-1.11/screens/didac-1440_900.mp4)

#### 1920x1080
[Image](https://openttd.github.io/title-screen-competition-1.11/screens/didac-1920_1080.png)
[Movie (low quality)](https://openttd.github.io/title-screen-competition-1.11/screens/didac-1920_1080.mp4)

#### 2560x1440
![poster](https://openttd.github.io/title-screen-competition-1.11/screens/didac-2560_1440.png)

[Image](https://openttd.github.io/title-screen-competition-1.11/screens/didac-2560_1440.png)
[Movie (low quality)](https://openttd.github.io/title-screen-competition-1.11/screens/didac-2560_1440.mp4)

**Reply by Kiki012184** [reply]
I like this one, im voting this

**Comment by 2TallTyler** [comment]
# Emperor Jake

There's a lot crammed onto a tiny map, the company is profitable and it's designed to look good at any resolution, from phones to high-resolution gaming monitors. Hope you like it!
#### 1024x768
![poster](https://openttd.github.io/title-screen-competition-1.11/screens/EmperorJake-1024_768.png)

[Image](https://openttd.github.io/title-screen-competition-1.11/screens/EmperorJake-1024_768.png)
[Animated GIF](https://openttd.github.io/title-screen-competition-1.11/screens/EmperorJake-1024_768.gif)
[Movie (low quality)](https://openttd.github.io/title-screen-competition-1.11/screens/EmperorJake-1024_768.mp4)

#### 1280x1024
[Image](https://openttd.github.io/title-screen-competition-1.11/screens/EmperorJake-1280_1024.png)
[Animated GIF](https://openttd.github.io/title-screen-competition-1.11/screens/EmperorJake-1280_1024.gif)
[Movie (low quality)](https://openttd.github.io/title-screen-competition-1.11/screens/EmperorJake-1280_1024.mp4)

#### 1440x900
[Image](https://openttd.github.io/title-screen-competition-1.11/screens/EmperorJake-1440_900.png)
[Animated GIF](https://openttd.github.io/title-screen-competition-1.11/screens/EmperorJake-1440_900.gif)
[Movie (low quality)](https://openttd.github.io/title-screen-competition-1.11/screens/EmperorJake-1440_900.mp4)

#### 1920x1080
[Image](https://openttd.github.io/title-screen-competition-1.11/screens/EmperorJake-1920_1080.png)
[Movie (low quality)](https://openttd.github.io/title-screen-competition-1.11/screens/EmperorJake-1920_1080.mp4)

#### 2560x1440
![poster](https://openttd.github.io/title-screen-competition-1.11/screens/EmperorJake-2560_1440.png)

[Image](https://openttd.github.io/title-screen-competition-1.11/screens/EmperorJake-2560_1440.png)
[Movie (low quality)](https://openttd.github.io/title-screen-competition-1.11/screens/EmperorJake-2560_1440.mp4)

**Comment by 2TallTyler** [comment, marked as answer]
# Chrnan6710

I'm hoping to redeem myself after my submission for last year's competition so tragically embarrassed itself in 1440p... also, I feel like we could all use some nice palm trees in our lives :)
#### 1024x768
![poster](https://openttd.github.io/title-screen-competition-1.11/screens/Chrnan6710-1024_768.png)

[Image](https://openttd.github.io/title-screen-competition-1.11/screens/Chrnan6710-1024_768.png)
[Animated GIF](https://openttd.github.io/title-screen-competition-1.11/screens/Chrnan6710-1024_768.gif)
[Movie (low quality)](https://openttd.github.io/title-screen-competition-1.11/screens/Chrnan6710-1024_768.mp4)

#### 1280x1024
[Image](https://openttd.github.io/title-screen-competition-1.11/screens/Chrnan6710-1280_1024.png)
[Animated GIF](https://openttd.github.io/title-screen-competition-1.11/screens/Chrnan6710-1280_1024.gif)
[Movie (low quality)](https://openttd.github.io/title-screen-competition-1.11/screens/Chrnan6710-1280_1024.mp4)

#### 1440x900
[Image](https://openttd.github.io/title-screen-competition-1.11/screens/Chrnan6710-1440_900.png)
[Animated GIF](https://openttd.github.io/title-screen-competition-1.11/screens/Chrnan6710-1440_900.gif)
[Movie (low quality)](https://openttd.github.io/title-screen-competition-1.11/screens/Chrnan6710-1440_900.mp4)

#### 1920x1080
[Image](https://openttd.github.io/title-screen-competition-1.11/screens/Chrnan6710-1920_1080.png)
[Movie (low quality)](https://openttd.github.io/title-screen-competition-1.11/screens/Chrnan6710-1920_1080.mp4)

#### 2560x1440
![poster](https://openttd.github.io/title-screen-competition-1.11/screens/Chrnan6710-2560_1440.png)

[Image](https://openttd.github.io/title-screen-competition-1.11/screens/Chrnan6710-2560_1440.png)
[Movie (low quality)](https://openttd.github.io/title-screen-competition-1.11/screens/Chrnan6710-2560_1440.mp4)

**Reply by aerottd** [reply]
This is hands down the best menu background proposal I've seen. You've certainly got my vote!

**Reply by Brandonlk** [reply]
Yay someone who actually uses monorail and boats :3

**Reply by holeholemax** [reply]
Love how parts are grass and parts are sand.

**Reply by MaxKorlaar** [reply]
This one is great!

**Reply by SecularSteve** [reply]
Not bad.
It has a little bit of everything.
Makes me wonder if there's really anything that's still missing from this game.
